ci/snapcraft.sh

```diff
@@ -0,0 +1,123 @@
+#!/bin/sh
+
+set -e
```

Comment:
suggest -u -f and -o pipefail as well

Reply:
Hmm, -u will render the checks at the top redundant or else they'll need rejiggering to cope if they're set to empty strings.  I guess -f would be harmless since there's no * or ? in the script anyway, so I'll add that to future-proof things.  This is a 'sh' so on the builders it'll end up with dash so -o pipefail isn't available according to that link.  I'll add -f and note that this passes shellcheck in case that mollifies you about the others.